prompterator/main.py

```diff
@@ -447,16 +447,7 @@ def _get_coloured_patch(patch):
     )
 
 
-def set_up_ui_labelling():
-    col1_orig, col2_orig = st.columns([1, 1])
-    text_orig_length = len(st.session_state.get("text_orig", ""))
-    col1_orig.text_area(
-        label=f"Original text ({text_orig_length} chars)",
-        key="text_orig",
-        disabled=True,
-        height=c.DATA_POINT_TEXT_AREA_HEIGHT,
-    )
-
+def set_up_prompt_atrs_area(col):
```

Comment:
Additionally, we could also rename `col` to something like `st_col` or even `st_container` (it is really a container according to the streamlit docs). This would be to prevent future confusion as we're also working a lot with table columns in the code.